### PR TITLE
Basic codelab for simple HTTP personalities

### PR DESCRIPTION
**Pull request by mhutchinson**
The POSIX and MySQL commands have been updated to output the required
env variables. I'll need some help with the GCP one as we can't assume
localhost and I don't think all the info we need is available. Maybe
this can be output by the terraform script?

### CODE REVIEW

File: cmd/conformance/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+This directory contains personalities that serve a dual purpose:
+ - To provide a simple example of a personality that can be deployed with each backend
+ - To function as conformance and performance harnesses for Tessera (using the [hammer](../../hammer/))
+
+Each subdirectory contains an implementation of the same personality built on top of Tessera.
+Implementations are provided that use:
+ - [A local filesystem](./posix/)
+ - [MySQL](./mysql/)
+ - [GCP](./gcp//)
```
Comment (roger2hk) [inline cmd/conformance/README.md:11]:
```suggestion
 - [GCP](./gcp/)
```

**Review by AlCutter — APPROVED** [review]

File: cmd/conformance/README.md (hunk 1 above)
Comment (mhutchinson) [reply cmd/conformance/README.md]: Good spot.

File: cmd/conformance/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Each of these personalities exposes an endpoint that accepts `POST` requests at a `/add` URL.
+The contents of any request body will be appended to the log, and the decimal index of the sequenced entry will be returned.
```
[the remark this answers is not in the document]
Comment (mhutchinson) [reply cmd/conformance/README.md]: Done.

File: cmd/conformance/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+## Codelab
+
+Choose one of the implementations above and deploy it.
+In the shell you are going to run this codelab in, define the following environment variables (check the logging output from starting the log, as these may have been output):
```
[the remark this answers is not in the document]
Comment (mhutchinson) [reply cmd/conformance/README.md]: Done.

**Pull request merged**